### PR TITLE
create writefile func

### PR DESCRIPTION
**Pull request by tausani-ah-chong**

### CODE REVIEW

File: utils.js [hunk 1]
```diff
@@ -17,3 +18,8 @@ function getViewData(filePath, callback) {
       }
     })
 }
+
+
+function writeNewData(filePath, petData) {
+  fs.writeFile(filePath, petData, 'utf8', () => {})
+}
```
Comment (sirchras) [inline utils.js:28]: my kingdom for a '\n'

**Review by sirchras — CHANGES_REQUESTED** [review]
Is this working?

**Comment by tausani-ah-chong** [comment]
updated callback

**Review by sirchras — APPROVED** [review]

**Comment by sirchras** [comment]
wish there was time to test

**Pull request merged**